tests/in/cubeArrayShadow.wgsl

```diff
@@ -0,0 +1,12 @@
+[[group(0), binding(4)]]
```

Comment:
can this be a part of `image.wgsl` instead?

Reply:
`image.wgsl` doesn't produce glsl because it uses 1D textures which aren't supported and `textureLoad` on depth textures which is also not supported.